### DISCUSSION TITLE
[Question]: 0.14 filmstrip with animated thumbnails for recent activity configurable ?

### DISCUSSION THREAD

**Discussion by 2Belette** [blakeblackshear/frigate#11857] [Ask A Question] [answered]
### What is your question:

Hi,

First of all : THANK YOU, I am using Frigate since 0.10 and this version is a game changer!

The idea of displaying the small filmstrip on top of live view is a very good idea, so good that I am wondering if this would be configurable for example to extend to the full day (or any thing else) instead of the last hour ?

I know there is "review" but it is also very convenient to get a quick look in live view and very efficient to scroll with the mouse..

Many thanks

**Comment by hawkeye217** [comment, marked as answer]
The filmstrip is not user-configurable at this point in time. Perhaps in a future update.

**Reply by RTCFlyer** [reply]
How's this future update coming? 

https://github.com/blakeblackshear/frigate/discussions/12699#discussioncomment-10209856

This is not really a solution
